en/flying/led_meanings.md

```diff
@@ -4,8 +4,13 @@
 
 ![LED meanings](../../images/led_meanings.gif)
 
-> **Note** **GPS Lock is required to takeoff: ** 
-  The LED will only turn green (ready to execute guided mission) if PX4 is able to validate the global position. If the LED does not change from blue to green, verify that the GPS module is properly attached, Pixhawk is reading your GPS properly, and that the GPS is sending a proper GPS position.
+> **Note** **A valid global position estimate is required to takeoff:** 
+  The LED will only turn green (ready to execute guided mission) if PX4 is able to validate the global position. If the LED does not change from blue to green, verify that the GPS module is properly attached, Pixhawk is reading your GPS properly, and that the GPS is sending a proper GPS position.  
+
+<span></span>
+> **Tip** In the event of an error (blinking red), or if the vehicle can't achieve GPS lock, check for more detailed status information in *QGroundControl*.
```

Comment:
@priseborough Would I be correct in thinking that if you get a preflight check it will display the blinking red LED?

Reply:
My understanding is that it should flash red if any preflight checks fail, but I have not tested that.